km_api/know_me/admin.py

```diff
@@ -124,6 +124,18 @@ class MediaResourceAdmin(admin.ModelAdmin):
     search_fields = ('name', 'km_user__user__name')
 
 
+@admin.register(models.ProfileAccessor)
+class ProfileAccessorAdmin(admin.ModelAdmin):
+    """
+    Admin for the ``ProfileAccessor`` model.
+    """
+    fields = ('km_user_accessor', 'profile', 'can_write')
+    list_display = ('km_user_accessor', 'profile', 'can_write')
+    search_fields = (
```

Comment:
Well since the `KMUserAccessor` doesn't have a `user` attribute, these won't work anyway :). I would suggest being able to search for the name of the profile owner as well as the name of the user with access.